tests/cache/tests.py

```diff
@@ -1185,9 +1185,8 @@ def test_invalid_keys(self):
     def test_memcached_uses_highest_pickle_version(self):
         # Regression test for #19810
         for cache_key, cache_config in settings.CACHES.items():
-            if cache_config['BACKEND'] == 'django.core.cache.backends.memcached.MemcachedCache':
-                self.assertEqual(caches[cache_key]._cache.pickleProtocol,
-                                 pickle.HIGHEST_PROTOCOL)
+            self.assertEqual(caches[cache_key]._cache.pickleProtocol,
```

Comment:
This test in python-memcached specific, so the skipUnless is still needed until the commit where the tests are refactored to use a base class + backend-specific classes.

I'm happy to fold this change into the refactoring commit however, if you prefer?
(If in doubt I split commits up to make it easier to review)